<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicineInfoDialog</class>
 <widget class="QDialog" name="MedicineInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>药品信息</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>361</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>药品名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ChemName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>    数   量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ChemAmt"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>药品序号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ChemNo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="add">
      <property name="text">
       <string>增</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="delete_">
      <property name="text">
       <string>删</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="change">
      <property name="text">
       <string>改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="search">
      <property name="text">
       <string>查</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
